cadasta/utilities/resources.py

```diff
@@ -79,6 +80,23 @@ def resource_url(path):
     return str(url.toString())
 
 
+def is_valid_url(url):
```

Comment:
Here's the unit test : https://github.com/dimasciput/cadasta-qgis-plugin/blob/project_creation/cadasta/utilities/test/test_resources.py#L40

I tried using QUrl.isValid() but both cases returns True.